app/helpers/patient_query_helper.rb

```diff
@@ -281,18 +281,52 @@ def advanced_filter(patients, filters, tz_offset)
         patients = advanced_filter_relative_date(patients, :symptom_onset, filter, tz_diff, :date)
       when 'continous-exposure'
         patients = patients.where(continuous_exposure: filter[:value].present? ? true : [nil, false])
+      when 'close-contact-with-known-case'
+        case filter[:additionalFilterOption]
+        when 'Exact Match'
+          patients = if filter[:value].blank?
+                       patients.where(contact_of_known_case_id: [nil, ''])
+                     else
+                       patients.where('patients.contact_of_known_case_id = ?', filter[:value])
```

Comment:
nope, will update

Reply:
It looks like your update wasn't actually the full change. It can just be: `patients.where(contact_of_known_case_id: filter[:value])`

Reply:
it was changed here: https://github.com/SaraAlert/SaraAlert/pull/701/commits/7af07a841d50b40941264675c542bd5892c90161. not sure why github isn't showing the full update

Reply:
Looking at that commit, what you changed it to is: 
``` patients.where('contact_of_known_case_id = ?', filter[:value])```

What it can be is:
```patients.where(contact_of_known_case_id: filter[:value])```

Reply:
ah I see - changed